src/components/PostList/PostList.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Post } from '../Post/Post';
+import './PostList.css';
+
+export const PostList = ({ posts, comments, users }) => {
+  const postListComplicate = posts.map(post => ({
```

Comment:
better to do this in App component so that when your data is changed, this component will rerender